extensions/amp-animation/amp-animation.md

```diff
@@ -25,278 +25,92 @@ limitations under the License.
 
 # amp-animation
 
-## Overview
+## Usage
 
-AMP Animations rely on [Web Animations API](https://www.w3.org/TR/web-animations/) to define and run animations in AMP documents.
+The `amp-animation` component defines and runs custom animations and effects. It relies on the [Web Animations API](https://www.w3.org/TR/web-animations/).
 
-## Format
-
-An `amp-animation` element defines such an animation as a JSON structure.
-
-### Top-level animation specification
-
-The top-level object defines an overall animation process which consists of an arbitrary number of animation components
-defined as an `animations` array:
+An `amp-animation` component defines animations in a JSON structure. The top-level section defines the overarching animation by declaring target element(s), execution conditions, timing properties and [keyframes effect](https://www.w3.org/TR/web-animations/#dom-keyframeeffect-keyframeeffect). The overarching process can contain any arbitrary number of animation parts defined within the `animations` array. Animation parts in the animation's array may have their own target elements, execution conditions, timing properties, and keyfreames effects.
 
 ```html
 <amp-animation layout="nodisplay">
   <script type="application/json">
     {
-      // Timing properties
-      ...
+      "selector": "#target-id",
+      "duration": "1s",
+      "iterations": "4",
+      "fill": "both",
+      "direction": "alternate",
       "animations": [
         {
-          // Animation 1
+          "media": "(max-width: 320px)",
+          "selector": ".target-class",
```

Comment:
I think we can also remove `selector` from here, at least from one of the parts. But not strictly necessary. But you do have a `selector` at the root level too, so one of these is extra.

Reply:
I removed one, but might be nice to demo you can effect multiple things in this way?

Reply:
Yeah. I think you're right. I'll leave the final decision with you on this.